tests/tests-tax.php

```diff
@@ -265,5 +265,18 @@ public function test_download_is_exclusive_of_tax() {
 	public function test_get_payment_tax() {
 		$this->assertEquals( '0.36', edd_get_payment_tax( $this->_payment_id ) );
 	}
+	
```

Comment:
This test is redundant to what we're already running here:
https://github.com/easydigitaldownloads/Easy-Digital-Downloads/blob/master/tests/tests-payments.php#L214